src/nvim/eval/userfunc.c

```diff
@@ -278,8 +278,8 @@ int get_lambda_tv(char_u **arg, typval_T *rettv, bool evaluate)
     ga_grow(&newlines, 1);
 
     // Add "return " before the expression.
-    len = 7 + e - s + 1;
-    p = (char_u *)xmalloc(len);
+    len = (int)(7 + e - s + 1);
```

Comment:
is len used after this as an `int`, otherwise just `size_t len = (size_t)(7 + e - s + 1);` here

Reply:
Good catch. Thanks.